src/test/java/se/liu/ida/hefquin/engine/data/impl/VocabularyMappingTest.java

```diff
@@ -48,58 +45,59 @@ public void VocabularyMappingConstructorTest() {
 	
 	@Test
 	public void TranslateTriplePatternTest() {
-		final Set<Triple> testTriples = CreateTestTriples();
+		final Pair<Set<Triple>,Set<org.apache.jena.graph.Triple>> testData= CreateTestTriples();
+		final Set<Triple> testTriples = testData.object1;
 		final VocabularyMapping vm = new VocabularyMappingImpl(testTriples);
 		
 		final Node s = NodeFactory.createLiteral("s1");
 		final Node p = NodeFactory.createURI(RDF.type.getURI());
 		final Node o = NodeFactory.createLiteral("o1");
 		final TriplePattern testTp = new TriplePatternImpl(s, p, o);
 		final SPARQLGraphPattern translation = vm.translateTriplePattern(testTp);
+		assert(translation instanceof SPARQLUnionPattern);
+		final Set<org.apache.jena.graph.Triple> translatedTriples = new HashSet<>();
+		for(final SPARQLGraphPattern i : ((SPARQLUnionPattern) translation).getSubPatterns()) {
+			assert(i instanceof TriplePattern);
```

Comment:
```suggestion
			assertTrue(i instanceof TriplePattern);
```
Please use the JUnit method to assert, also in line 57